source/administration/production-notes.txt

```diff
@@ -559,6 +559,43 @@ possible, use XFS as it generally performs better with MongoDB.
    *on directories*. For example, HGFS and Virtual Box's shared
    folders do *not* support this operation.
 
+Set ``vm.swappiness`` at least ``1``
+````````````````````````````````````
+
+“Swappiness” is a Linux kernel setting that influences the behavior of
+the Virtual Memory manager when it needs to allocate a swap, ranging
+from ``0`` to ``100``, inclusive. 
+
+- A setting of ``0`` tells the kernel to swap only to avoid
+  out-of-memory problems. 
+
+- A setting of ``100`` tells it to swap aggressively to disk.
+
+If your host runs kernel versions ``3.5`` or later, or ``2.6.32-303`` 
+or later, setting this value to ``0`` could disable swapping. Set this 
```

Comment:
I don't understand this qualification. Why not just say "kernel version 2.6.32-303 or later"? Are there some versions between 2.6.32-303  and 3.5 to which this doesn't apply?

Reply:
I clarified.